dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -130,6 +132,17 @@ UpdateLanStatus(HWND hwndDlg,  LANSTATUSUI_CONTEXT * pContext)
         return;
     }
 
+    if(bFirst)
```

Comment:
I see that you can determine whether you're in "first" case by just looking at the `pContext->Status` value, that conveniently has the value `-1` at the first call (see where you set bFirst to TRUE below).
This would allow not using this ugly global variable ^^

Reply:
Nice idea ! I'll do it this way